tests/infrastructure/instance_types/test_common_vm_preference.py

```diff
@@ -132,32 +76,42 @@ def test_common_preferences_vendor_labels(base_vm_cluster_preferences):
 class TestCommonVmPreference:
     @pytest.mark.polarion("CNV-9894")
     def test_common_vm_preference_windows(self, unprivileged_client, namespace):
-        run_general_vm_preferences(client=unprivileged_client, namespace=namespace, preferences=WINDOWS_PREFERENCE_LIST)
+        run_general_vm_preferences(
+            client=unprivileged_client, namespace=namespace, preferences=VM_PREFERENCES_LIST["windows"]
+        )
```

Comment:
why separate tests for windows and linux? can't this one be a param in `test_common_vm_preference_linux`?

Reply:
agreed. thanks!